lib/example_17x_app_web/views/thing_view.ex

```diff
@@ -1,5 +1,4 @@
 defmodule Example17xAppWeb.ThingView do
-  use Example17xAppWeb, :view
```

Comment:
We can't use `use Example17xAppWeb, :view` (well, not without creating it) as phoenix has moved away from this idea with the move to phoenix 1.7.X.